clusterloader2/testing/load/config.yaml

```diff
@@ -255,7 +257,8 @@ steps:
       actionName: "scale and update"
       namespaces: {{$namespaces}}
       tuningSet: RandomizedScalingTimeLimited
-      randomScaleFactor: 0.5
+      randomScaleFactor: {{$RANDOM_SCALE_FACTOR}}
+      testMaxReplicaFactor: {{$RANDOM_SCALE_FACTOR}}
```

Comment:
It's very hard to tell what it does based on the name. Do we really need to expose this to top level of the config?

Reply:
Not sure what other alternative we have - we could use constant, however I do not like the idea that this constant will silently fail based on some other change.

Reply:
Ohh,.. Now I understand what it means.

How about using "testMaxRandomScaleFactor" name or something like that? I initially didn't even now that it refers to maximum allowable value for randomScaleFactor